src/content/en/tools/workbox/v3/guides/precache-files.md

```diff
@@ -0,0 +1,290 @@
+project_path: /web/tools/workbox/v3/_project.yaml
+book_path: /web/tools/workbox/v3/_book.yaml
+description: A guide on how to precache files with Workbox.
+
+{# wf_updated_on: 2017-11-15 #}
+{# wf_published_on: 2017-11-15 #}
+
+# Precache Files {: .page-title }
+
+{% include "web/tools/workbox/v3/_shared/alpha.html" %}
+
+If you want your web app to work offline or there are assets you know can be
+cached for a long time, precaching is the best approach.
+
+Precaching a file will ensure that a file is downloaded and cached before a
+service worker is installed, meaning that if you service worker is installed,
+your files will be cached.
+
+Workbox provides an easy way to precache files, ensuring that as your service
+worker changes, the precached files are maintained efficiently, only
+downloaded updated files and cleaning up after the service worker is made
+redundant.
+
+```javascript
```

Comment:
This makes it seem like the developer manually creates the manifest array, which was a huge GOTCHA for me when first learning Workbox

Reply:
No action needed ATM, just noting